docs/OV_Runtime_UG/integrate_with_your_application.md

```diff
@@ -12,10 +12,10 @@
 
 @endsphinxdirective
 
```

Comment:
This article has an updated version by Karol in master. You may need to fetch it from master first. @sgolebiewski-intel

Reply:
It's been merged :)

Reply:
It's still different with the version on master...